### PR TITLE
Issue 1

### PR DESCRIPTION
**Pull request by Casey1449**
@stevekinney @martensonbj @brittanystoroz @tehviking

## Purpose

[issue #1](https://github.com/turingschool-projects/1606-remember-2/issues/1)

#1 

Takes user to the reminder-list as the root path, renders the reminders.

## Approach

We set the path to the 'reminder-list' template as '/' in our router.js file. Within the 'reminder-list' template, we 'each' through the reminders provided by the model, and inject them into a 'reminder-item' component. This component extracts and formats the reminder title, date, and body, and renders them in the context of a list-item, with the proper class name.

### Learning

We leaned heavily on the grocery-list repo to approach this task. Also, Brendan helped us understand how a template's 'route' file is the proper place for shoveling data from the model to the markup, by using Ember's 'findAll' to return an array of model objects.

### Open Questions and Pre-Merge TODOs

- [ ] Use Github checklists. When solved, check the box and explain the answer.

### Test coverage 

Tests were provided.

### Merge Dependencies and Related Work

_Link to merge dependency issues or PRs._

### Follow-up tasks

Try to get the 'click each reminder' tests to pass.

### Screenshots

image

#### After

### CODE REVIEW

File: app/templates/components/reminder-item.hbs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<li class="spec-reminder-item">
```
Comment (stevekinney) [inline app/templates/components/reminder-item.hbs]: It's an interesting choice to have a bunch of `<li>`. Maybe we can do something more semantic like putting the title in a `<h2>` or `<h3>` then wrapping the other information in `<p>` or `<div>` tags.

File: app/models/reminder.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  date: DS.attr('date'),
+  notes: DS.attr('string'),
+  pinned: DS.attr('boolean')
```
Comment (stevekinney) [inline app/models/reminder.js]: Can you default this to `false`?

File: issues.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+## 1 [x]
```
Comment (stevekinney) [inline issues.md]: Let's not check this into version control. Please remove it.

File: package.json [hunk 4]
```diff
@@ -40,6 +40,7 @@
     "ember-load-initializers": "^0.5.1",
     "ember-resolver": "^2.0.3",
     "ember-welcome-page": "^1.0.1",
-    "loader.js": "^4.0.1"
+    "loader.js": "^4.0.1",
+    "phantomjs": "^2.1.7"
```
Comment (stevekinney) [inline package.json]: Why were these added?

Comment (Casey1449) [inline package.json]: We got an error during setup for not having phantomjs. I can't remember if it was the npm install, bower install, or the 'brew install watchman' (which we did to remove the warning about not having watchman)...should we delete from package.json?

File: issues.md (hunk 3 above)
Comment (Casey1449) [inline issues.md]: You mean this whole file, I'm guessing?

Comment (stevekinney) [inline issues.md]: Yea.

File: package.json (hunk 4 above)
Comment (martensonbj) [inline package.json]: I'd imagine your phantomjs error was because you were missing the dependency globally, not project specific - run `npm i -g phantomjs` if you haven't already and I bet you don't need lines 43/44.

**Comment by stevekinney** [comment]
Just checking in to see if you pushed the changes you made with me and @tehviking today.

File: app/templates/reminder-list.hbs [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<h1>Reminders: </h1>
+<ul class="reminder-list">
+
+    {{#each model as |reminder|}}
+      {{reminder-item reminder=reminder}}
+    {{/each}}
+
+</ul>
```
Comment (martensonbj) [inline app/templates/reminder-list.hbs]: Similar to the comment from Steve above, but if you haven't already make sure this gets changed to a more semantically appropriate element. (`<section>` & `<article>` perhaps?)

File: app/models/reminder.js (hunk 2 above)
Comment (blakeworsley) [inline app/models/reminder.js]: Fixed in upcoming PR:
screen shot 2016-10-13 at 9 46 26 am

File: issues.md (hunk 3 above)
Comment (blakeworsley) [inline issues.md]: Removed in upcoming issue-2 PR

File: package.json (hunk 4 above)
Comment (blakeworsley) [inline package.json]: Removed Phantom.js and installed locally on both machines in upcoming issue-2 PR

File: app/templates/reminder-list.hbs (hunk 5 above)
Comment (blakeworsley) [inline app/templates/reminder-list.hbs]: Discussed in person with Steve and Brandon.

File: app/templates/components/reminder-item.hbs (hunk 1 above)
Comment (blakeworsley) [inline app/templates/components/reminder-item.hbs]: Discussed in person.

**Comment by brittanystoroz** [comment]
Can we change the name of this pull request? It was really confusing to see it labeled "Issue 1" and realize that I was looking at a pull request rather than an issue. PR titles should be similar to a commit message in that it tells you (very briefly) what the PR is for. You are already linking to the issue it reports to in the purpose section of the description, which is plenty.

**Comment by stevekinney** [comment]
Addressed in #6
